docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -325,16 +329,134 @@ When the proxy datasource is obtained via a JNDI lookup by the JDBC persistence
+    <dataSource id="fhirDatasourceDefaultDefault" jndiName="jdbc/fhir_default_default" type="javax.sql.XADataSource" statementCacheSize="200" syncQueryTimeoutWithTransactionTimeout="true">
+        <jdbcDriver javax.sql.XADataSource="com.ibm.db2.jcc.DB2XADataSource" libraryRef="fhirSharedLib"/>
+            <properties.db2.jcc
+                 serverName="db2"
+                 portNumber="*****"
+                 user="*****"
+                 password="*****"
+                 databaseName="*****"
+                 currentSchema="*****"
+                 driverType="4"
+             />
+        />
+        <connectionManager maxPoolSize="200" minPoolSize="40"/>
+    </dataSource>
+</server>
+```
+
+This file is picked up when the server starts as indicated by the following AUDIT message:
+
+```
+[AUDIT   ] CWWKG0093A: Processing configuration drop-ins resource: /fhir/wlp/usr/servers/fhir-server/configDropins/overrides/datasource-default.xml
+```
+
+A special case exists for configuring the Derby 'bootstrapped' datasources. To avoid naming conflicts, these datasources must use custom JNDI names, not the default naming pattern. For example, a Derby datasource defined in configDropins/overrides/datasource-bootstrap.xml' might look like this:
+
+```
+<server>
+    <dataSource id="fhirDatasourceBootstrapDefaultDefault" jndiName="jdbc/bootstrap_default_default" type="javax.sql.XADataSource" statementCacheSize="200" syncQueryTimeoutWithTransactionTimeout="true">
+        <jdbcDriver javax.sql.XADataSource="org.apache.derby.jdbc.EmbeddedXADataSource" libraryRef="fhirSharedLib"/>
+            <properties.derby.embedded createDatabase="create" databaseName="derby/fhirDB"/>
+        <connectionManager maxPoolSize="50" minPoolSize="10"/>
+    </dataSource>
+</server>
+```
+
+Note the use of 'jdbc/bootstrap'. How this is used is described below. The corresponding definition in 'fhir-server-config.json' must be configured like this:
+
+```
+    {
+        "fhirServer":{
+
+            "persistence": {
+                ...
+                "jdbc": {
+                    "bootstrapDb": true,
+                    "enableProxyDatasource": false,
+                    "bootstrapDataSourceBase": "jdbc/bootstrap"
+                    ...
+                },
+                "datasources": {
+                    "default": {
+                        "jndiName": "jdbc/bootstrap_default_default",
+                        "type": "derby",
+                        "currentSchema": "APP"
+                    },
+                    ...
+                }
+            }
+        }
+    }
+```
+
+Note 1. The introduction of the new property 'bootstrapDataSourceBase'. This value is required to correctly identify the datasources to use when bootstrapping the Derby databases. The value 'jdbc/bootstrap' is the prefix which should be common to all the Derby datasource JNDI names referencing bootstrapped Derby databases (currently 'default', 'profile', 'reference', 'study1').
```

Comment:
Curious as to why boostrapDataSourceBase? what does it mean? 
If it's just a lookup name, maybe be a bit more direct
`bootstrapDataSourceBase` maybe simplify to bootstrapName ...

Reply:
or make it look like `dataSourceJndiName` in one of the examples

Reply:
It's a prefix string acting as the base name for a number of actual JNDI names. It's a quirk related to how the current bootstrap code works (which ideally should be refactored, but won't be doing that here - already covered in another issue not planned for this sprint).